src/Database/Database.php

```
@@ -3565,6 +3566,503 @@ private function populateDocumentRelationships(Document $collection, Document $d
+            return $attribute->getAttribute('type') === Database::VAR_RELATIONSHIP;
+        });
+
+        if (empty($relationships)) {
+            return $documents;
+        }
+
+        foreach ($relationships as $relationship) {
+            $key = $relationship['key'];
+            $relatedCollection = $this->getCollection($relationship['options']['relatedCollection']);
+            $relationType = $relationship['options']['relationType'];
+            $twoWay = $relationship['options']['twoWay'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$twoWay'. (Unused Code Rules)

(UnusedLocalVariable)